README.md

```diff
@@ -82,13 +82,20 @@ GOOS=linux GOARCH=arm64 go build -ldflags "-s -w" -o xray cmd/tracing/daemon.go
 As of Aug 31, 2020, windows and darwin builds for ARM64 are not supported by `go build`.
 
 ## Pulling X-Ray Daemon image from ECR Public Gallery
+Before pulling an image authenticate your docker client to the Amazon ECR public registry. For registry authentication options follow this [link](https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html#public-registry-auth)
```

Comment:
I think "you should authenticate" - I guess it still works without, just with low quota.

Reply:
Not sure if pulling without an authentication works as per ECR doc.(https://docs.aws.amazon.com/AmazonECR/latest/public/docker-pull-ecr-image.html). As per my understanding you have to authenticate docker client with public ECR in order to use `docker pull` command

Reply:
Hmm the docs aren't so clear but the blog posts have all mentioned anonymous pull with a smaller quota

https://aws.amazon.com/blogs/aws/amazon-ecr-public-a-new-public-container-registry/

```
Anyone who pulls images anonymously will get 500 GB of free data bandwidth each month, after which they can sign up or sign in to an AWS account to get more. Simply authenticating with an AWS account increases free data bandwidth up to 5 TB each month when pulling images from the internet.
```

Reply:
I see. Nice!